chore: Add configuring nested containers article (#2986)
codenvy/che-docs: modules/administration-guide/pages/enabling-container-run-capabilities.adoc

:_content-type: PROCEDURE
:description: Enabling container run capabilities
:keywords: administration-guide, container, podman, nested containers
:navtitle: Enabling container run capabilities
:page-aliases:

[id="enabling-container-run-capabilities"]
= Enabling container run capabilities

You can enable container run capabilities in {prod-short} workspaces to
allow running nested containers using tools like Podman. This feature leverages Linux kernel

> user namespaces for isolation, enabling users to build and run container

Reviewer: ```suggestion
user namespaces for isolation, so that users can build and run container
```

images within their workspaces.

[IMPORTANT]
====

> * Previously created workspaces cannot be started after enabling this feature. Users will need to create new workspaces.

Reviewer: ```suggestion
Previously created workspaces can not be started after enabling this feature. Users will need to create new workspaces.
```
